src/WhatToCook.WebApp/Controllers/MealPlanningController.cs

```diff
@@ -38,9 +32,20 @@ public async Task<ActionResult> Post(PlanOfMealRequest planOfMealRequest)
         return Ok();
     }
     [HttpPut]
-    public async Task<ActionResult>Put(UpdatePlanOfMealRequest planOfMealRequest)
+    public async Task<ActionResult> Put(UpdatePlanOfMealRequest planOfMealRequest)
     {
         await _mealPlanningService.Update(planOfMealRequest);
         return Ok();
     }
+    [HttpGet("GetIngredients/{mealPlanId}")]
```

Comment:
Maybe, let's make a URL same as functionality name - GetShoppingList, isn't that a little bit more descriptive?